artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PageTransactionInfoImpl.java

```diff
@@ -67,11 +67,27 @@ public final class PageTransactionInfoImpl implements PageTransactionInfo {
 
    private List<LateDelivery> lateDeliveries;
 
+   /** To be used during by the RebuildManager.
+    *  When reading transactions not found transactions are marked as done. */
+   private boolean orphaned;
+
+
```

Comment:
Extra newline added, seems like some extras were removed down below so probably want to prune this one